lib/Conversion/TorchToMhlo/Reduction.cpp

```diff
@@ -85,17 +85,22 @@ getMaxInDim(ConversionPatternRewriter &rewriter, Operation *op, Value &input,
 
   Value initValue = createInitialValueForReduceOp(op, inputElemTy, rewriter);
   if (!initValue) return llvm::None;
-
-  Value initIndex =
+  Value initIndex;
+  if (mlir::mhlo::kMhloDimSizeBits == 32) {
+    initIndex =
+      mhlo::getConstTensor<int64_t>(rewriter, op, {0}, {}).getValue();
```

Comment:
Any problems here? The contents in `if` branch and `else` branch is exactly same.

Reply:
`mhlo::getConstTensor<int64_t> ` emits `%6 = mhlo.constant dense<0> : tensor<64>`, but `mhlo::getConstTensor<int32_t>` emits `%6 = mhlo.constant dense<0> : tensor<i32>`

Reply:
It's better to have some unit tests to check the i32 shape, and I left an issue #1191